packages/wonder-blocks-modal/components/modal-panel.js

```diff
@@ -60,6 +60,21 @@ type Props = {|
     onClose?: () => mixed,
 |};
 
+/**
+ * ModalPanel is  the content container.
+ *
+ * **Implementation notes:**
+ *
+ * If you are creating a custom Dialog, make sure to follow these guidelines:
+ * - Make sure to add this component inside the **ModalDialog**.
```

Comment:
It would be nice if there was a way to cross link to `ModalDialog`.  No action necessary for this PR, but maybe something to look at for the docs in general.

Reply:
Good point! I'm going to add a quick fix in the meantime. But it would be great to revisit that in general for sure!